mmv1/products/accesscontextmanager/terraform.yaml

```diff
@@ -26,11 +26,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     skip_sweeper: true
     id_format: "{{name}}"
     import_format: ["{{name}}"]
+    iam_policy: !ruby/object:Api::Resource::IamPolicy
```

Comment:
Do the IAM & scope change need to happen simultaneously, or can they be broken up? Smaller, focused PRs are generally preferable & reduce the blasting radius of cherrypicks (by customers) and/or reverts (by us).

Reply:
As long as they can be included in the same release, I have no problem splitting them up.

Reply:
Barring a mixup they'll go out in the same release- if there's a strong need to ensure they're in _exactly_ the same one, keeping them together makes sense though.

Reply:
In that case, it would make sense to keep them together. They are very tightly coupled.